etcdserver/raft.go

```diff
@@ -226,6 +226,10 @@ func (r *raftNode) start(s *EtcdServer) {
 				if err := r.storage.Save(rd.HardState, rd.Entries); err != nil {
 					plog.Fatalf("raft save state and entries error: %v", err)
 				}
+				if !raft.IsEmptyHardState(rd.HardState) {
+					proposalsCommitted.Set(float64(rd.HardState.Commit))
```

Comment:
@fabxc I do not think this is good as suggested in the godoc of prom's go-client... But on the other hand, it is very inconvenient to calculate the diff...